paper-dialog-behavior.html

```diff
@@ -140,12 +140,13 @@
       } else {
         this.setAttribute('aria-modal', 'false');
       }
-      // modal implies noCancelOnOutsideClick and withBackdrop if true, don't overwrite
-      // those properties otherwise.
```

Comment:
These properties are declared in `IronOverlayBehavior` (ex: https://github.com/PolymerElements/paper-dialog-behavior). It may be that since `modal` is changing before `ready` is called, the attributes have not been deserialized to property values yet. However, it is not good to set the values on `this` in this condition, because if you do it too early then Polymer does not create a setter / getter for data-binding (I think).